bin/import.js

```diff
@@ -55,14 +57,30 @@ function handleImport (args) {
 
   openDat(args, function (err, db) {
     if (err) abort(err, args)
-    doImport(db)
+    if (args._[0] === '-') doImport(process.stdin, db)
+    else {
+      getReadStream(args._[0], function (err, inputStream) {
+        if (err) abort(err, args, err.message)
+        doImport(inputStream, db)
+      })
+    }
   })
 
-  function doImport (db) {
-    var inputStream
-    if (args._[0] === '-') inputStream = process.stdin
-    else inputStream = fs.createReadStream(args._[0])
+  function getReadStream (location, cb) {
+    fs.exists(location, function (exists) {
+      if (exists) {
+        cb(null, fs.createReadStream(location))
+      } else {
+        var u = url.parse(location)
+        if (u.protocol && u.protocol.match(/http/)) {
+          cb(null, got(location))
+        }
+        else cb(new Error('File location does not exist.'))
```

Comment:
i think the style he was referring to was the mixed brackets

```
if (foo) { baz() }
else bar()
```

usually its all one or the other, e.g.

```
if (foo) { baz() }
else { bar () }

if (foo) baz()
else bar()
```

but standard doesnt seem to care